pkg/transformers/labelsandannotations_test.go

```diff
@@ -36,6 +36,7 @@ var crd = schema.GroupVersionKind{Group: "apiwctensions.k8s.io", Version: "v1bet
 var job = schema.GroupVersionKind{Group: "batch", Version: "v1", Kind: "Job"}
 var cronjob = schema.GroupVersionKind{Group: "batch", Version: "v1beta1", Kind: "CronJob"}
 var pvc = schema.GroupVersionKind{Version: "v1", Kind: "PersistentVolumeClaim"}
+var rb = schema.GroupVersionKind{Version: "rbac.authorization.k8s.io/v1", Kind: "ClusterRoleBinding"}
```

Comment:
A small nit -- it looks like the convention so far has been to name these variables after the resource short name, which in this case is `clusterrolebinding`.